dir.props

```diff
@@ -45,19 +45,45 @@
     <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
   </PropertyGroup>
   
+  <!-- 
+  Projects that have no OS-specific implementations just use Debug and Release for $(Configuration).
+  Projects that do have OS-specific implementations use OS_Debug and OS_Release, e.g. a project with
+  a Windows implementation and a Unix implementation will have Windows_Debug, Windows_Release, Unix_Debug,
+  and Unix_Release.  For cases where a general Unix implementation needs to be further specialized for Linux vs Mac,
+  a project may have Windows, Linux, and Mac implementations, in which case it would have Windows_Debug,
+  Windows_Release, Linux_Debug, Linux_Release, Mac_Debug, and Mac_Release.
+  
+  Since now have multiple *Debug and *Release configurations, ConfigurationGroup is set to Debug for any of the
+  debug configurations, and to Release for any of the release configurations.
+  -->
+  
   <!-- Setup Default symbol and optimization for Configuration -->
-  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug' or '$(Configuration)' == 'Windows_Debug' or '$(Configuration)' == 'Linux_Debug' or '$(Configuration)' == 'Mac_Debug' or '$(Configuration)' == 'Unix_Debug'">
```

Comment:
This condition can be simplified to '$(Configuration.EndsWith('Debug'))'. The same can be done with the Release Check.